> Can you update to the latest develop branch - this should fix the CI - especially Clang-Tidy.

Now the only remaining CI errors are:

```
FAILED: tests/CMakeFiles/test-conversions_cpp11.dir/src/unit-conversions.cpp.o 
/__w/_temp/447077027/cmake-3.31.0-linux-x86_64/bin/cmake -E __run_co_compile --tidy="/usr/bin/clang-tidy-20;--extra-arg-before=--driver-mode=g++" --source=/__w/json/json/tests/src/unit-conversions.cpp -- /usr/bin/clang++ -DDOCTEST_CONFIG_SUPER_FAST_ASSERTS -DJSON_TEST_KEEP_MACROS -DJSON_TEST_USING_MULTIPLE_HEADERS=1 -I/__w/json/json/tests/thirdparty/doctest -I/__w/json/json/tests/thirdparty/fifo_map -I/__w/json/json/build/build_clang_tidy/include -I/__w/json/json/include -g -std=gnu++11 -Wno-deprecated -Wno-float-equal -MD -MT tests/CMakeFiles/test-conversions_cpp11.dir/src/unit-conversions.cpp.o -MF tests/CMakeFiles/test-conversions_cpp11.dir/src/unit-conversions.cpp.o.d -o tests/CMakeFiles/test-conversions_cpp11.dir/src/unit-conversions.cpp.o -c /__w/json/json/tests/src/unit-conversions.cpp
/__w/json/json/tests/src/unit-conversions.cpp:384:19: error: do not declare C-style arrays, use 'std::array' instead [cppcoreguidelines-avoid-c-arrays,hicpp-avoid-c-arrays,modernize-avoid-c-arrays,-warnings-as-errors]
  384 |             const int nbs[][2][3] = {\
      |                   ^
/__w/json/json/tests/src/unit-conversions.cpp:388:13: error: do not declare C-style arrays, use 'std::array' instead [cppcoreguidelines-avoid-c-arrays,hicpp-avoid-c-arrays,modernize-avoid-c-arrays,-warnings-as-errors]
  388 |             int nbs2[][2][3] = {\
      |             ^
/__w/json/json/tests/src/unit-conversions.cpp:400:19: error: do not declare C-style arrays, use 'std::array' instead [cppcoreguidelines-avoid-c-arrays,hicpp-avoid-c-arrays,modernize-avoid-c-arrays,-warnings-as-errors]
  400 |             const int nbs[][2][2][3] = {\
      |                   ^
/__w/json/json/tests/src/unit-conversions.cpp:412:13: error: do not declare C-style arrays, use 'std::array' instead [cppcoreguidelines-avoid-c-arrays,hicpp-avoid-c-arrays,modernize-avoid-c-arrays,-warnings-as-errors]
  412 |             int nbs2[][2][2][3] = {\
      |             ^
90313 warnings generated.
Suppressed 90618 warnings (90271 in non-user code, 347 NOLINT).
Use -header-filter=.* to display errors from all non-system headers. Use -system-headers to display errors from system headers as well.
4 warnings treated as errors
```

I'm not sure how to proceed. The purpose of this change is to support C-style arrays, which contradicts the clang-tidy rules of avoiding them. What is your suggestion? @nlohmann

--- a/tests/src/unit-conversions.cpp
+++ b/tests/src/unit-conversions.cpp
@@ -381,14 +381,16 @@ TEST_CASE("value conversion")
 
         SECTION("built-in arrays: 3D")
         {
+            // NOLINTBEGIN(cppcoreguidelines-avoid-c-arrays,hicpp-avoid-c-arrays,modernize-avoid-c-arrays)
             const int nbs[][2][3] = {\
                 {{0, 1, 2}, {3, 4, 5}}, \
                 {{10, 11, 12}, {13, 14, 15}}\
-            }; // NOLINT(cppcoreguidelines-avoid-c-arrays,hicpp-avoid-c-arrays,modernize-avoid-c-arrays)
+            };
             int nbs2[][2][3] = {\
                 {{0, 0, 0}, {0, 0, 0}}, \
                 {{0, 0, 0}, {0, 0, 0}}\
-            }; // NOLINT(cppcoreguidelines-avoid-c-arrays,hicpp-avoid-c-arrays,modernize-avoid-c-arrays)
+            };
+            // NOLINTEND(cppcoreguidelines-avoid-c-arrays,hicpp-avoid-c-arrays,modernize-avoid-c-arrays)
 
             const json j2 = nbs;
             j2.get_to(nbs2);
@@ -397,6 +399,7 @@ TEST_CASE("value conversion")
 
         SECTION("built-in arrays: 4D")
         {
+            // NOLINTBEGIN(cppcoreguidelines-avoid-c-arrays,hicpp-avoid-c-arrays,modernize-avoid-c-arrays)
             const int nbs[][2][2][3] = {\
                 {
                     \
@@ -408,7 +411,7 @@ TEST_CASE("value conversion")
                     {{20, 21, 22}, {23, 24, 25}}, \
                     {{30, 31, 32}, {33, 34, 35}}\
                 }\
-            }; // NOLINT(cppcoreguidelines-avoid-c-arrays,hicpp-avoid-c-arrays,modernize-avoid-c-arrays)
+            };
             int nbs2[][2][2][3] = {\
                 {
                     \
@@ -420,7 +423,8 @@ TEST_CASE("value conversion")
                     {{0, 0, 0}, {0, 0, 0}}, \
                     {{0, 0, 0}, {0, 0, 0}}\
                 }\
-            }; // NOLINT(cppcoreguidelines-avoid-c-arrays,hicpp-avoid-c-arrays,modernize-avoid-c-arrays)
+            };
+            // NOLINTEND(cppcoreguidelines-avoid-c-arrays,hicpp-avoid-c-arrays,modernize-avoid-c-arrays)
 
             const json j2 = nbs;
             j2.get_to(nbs2);

